Projects/Offer/Sources/Offer.swift

```diff
@@ -209,7 +209,11 @@ class OfferState {
 		return client.perform(mutation: GraphQL.SignQuotesMutation(ids: ids))
 			.map { data in
 				if data.signQuotes.asAlreadyCompleted != nil {
-					self.$hasSignedQuotes.value = true
+					Future().delay(by: 1)
```

Comment:
What's the delay for here?

Reply:
Because if signing was too fast it felt like something went wrong 😅

Reply:
Buut, thinking about it signing in prod might be slow enough already 🤔

Reply:
Can't this introduce race conditions tho?

Reply:
Yeah I don't think we should introduce that much delay tho if we don't need it. Maybe 0.5 is ok.